autogluon/utils/tabular/ml/models/catboost/catboost_model.py

```diff
@@ -73,14 +72,15 @@ def fit(self, X_train, Y_train, X_test=None, Y_test=None, time_limit=None, **kwa
             num_classes = 1
 
         # TODO: Add ignore_memory_limits param to disable NotEnoughMemoryError Exceptions
+        max_memory_usage_ratio = self.params_aux['max_memory_usage_ratio']
```

Comment:
should value of this be reduced to mitigate those ngram Catboost errors?

Reply:
Those errors were due to multiple notebooks being opened at once, so it wasn't actually the fault of CatBoost. More generally, we want max_memory_usage_ratio=1 to mean that the model is safe assuming AG is the only major process running. What 1 means is determined by the model, as we would want to reduce those internal values rather than max_memory_usage_ratio to ensure 1.0 is safe.